types_generate.go

```diff
@@ -272,6 +277,12 @@ func main() {
 					fields = append(fields, f)
 					continue
 				}
+				if t == "SVCBKeyValue" {
+					fmt.Fprintf(b, "%s := make([]%s, len(rr.%s));\nfor i,e := range rr.%s {\n %s[i] = e.copy()\n}\n",
```

Comment:
You can drop the `\n`s from here, it's formatted anyway.

Reply:
It needs to be consistent with the previous 8 lines of code?

Reply:
That's fine then.